server/views/pages/statement/your-statement.html

```diff
@@ -60,17 +60,58 @@ <h1 class="govuk-heading-xl mainHeading">Your use of force statement</h1>
   </div>
   <hr />
 
-  <div class="govuk-grid-row">
-    <div class="govuk-grid-column-full">
-      {{
-        govukButton({
-          text: 'Return to use of force incidents',
-          href: '/',
-          classes: 'govuk-button govuk-!-margin-top-5',
-          attributes: { 'data-qa': 'cancel' }
-        })
-      }}
+  {% for comment in data.additionalComments %}
+    <div class="govuk-grid-row govuk-!-margin-top-4">
+      <div class="govuk-grid-column-one-half">
+        <p class="govuk-!-margin-bottom-0 govuk-label--s">Additional comment</p>
+      </div>
+      <div class="govuk-grid-column-one-half ">
+        <p>
+          <span class="govuk-label--s">Submitted:</span> {{ comment.dateSubmitted | formatDate('DD MMMM YYYY - HH:mm') }}
+        </p>
+      </div>
     </div>
-  </div>
+  {{comment.additionalComment}}<hr/>
+  {% endfor %}
+
+  <form method="POST">
+  
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+        {{
+          govukTextarea({
+            id: 'additionalContent',
+            name: 'additionalContent',
+            rows: 15,
+            cols: 200,
+            label: {
+              text: 'Additional comments',
+              classes: 'govuk-label--m govuk-!-margin-top-4'
+            }          
+          })
+        }}
+      </div>
+    </div>
+   
+    {{
+      govukButton({
+        text: 'Save additional comments',
+        name: 'saveAdditionalComments',
+        value: 'save-and-continue',
+        classes: 'govuk-button',
+        attributes: { 'data-qa': 'save-and-continue' }
+      })
+    }}
+    {{
+      govukButton({
+        text: 'Return to use of force statements',
```

Comment:
I think this can just be the same as it was before and just be a link back to the main statement page.

I don't think we will want to support an amendment lifecycle and support temporarily saving it before submitting? it would be confusing how to display that there is an incomplete amendment on the statement page.

Reply:
updated as per comments, plus addition of     
<input type="hidden" name="_csrf" value="{{ csrfToken }}" />